### PR TITLE
[17.0][MIG] account_financial_risk

### PR DESCRIPTION
**Pull request by sbiosca-s73**
Migration module to v17.

The module is migrated in this PR https://github.com/OCA/credit-control/pull/347, but we have not received a response and we need it the migration correctly.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check CI.

/ocabot migration account_financial_risk

File: account_financial_risk/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+        )
+        customers = self.filtered(
+            lambda p: p == p.commercial_partner_id
+            or (p._origin and p._origin.id in p.commercial_partner_id.ids)
+        )
+        if not customers:
+            return  # pragma: no cover
+        groups = self._risk_account_groups()
+        for _key, group in groups.items():
+            group["read_group"] = self.env["account.move.line"].read_group(
```
Comment (carlosdauden) [inline account_financial_risk/models/res_partner.py]: Use _read_group, same that:
https://github.com/OCA/credit-control/pull/377#discussion_r1587498744

**Review by carlosdauden — CHANGES_REQUESTED** [review]

File: account_financial_risk/models/res_partner.py (hunk 1 above)
Comment (sbiosca-s73) [reply account_financial_risk/models/res_partner.py]: Yes, I had it in mind, but it passed me.
Changes done.

**Comment by sbiosca-s73** [comment]
@carlosdauden tests correctly

**Review by ProcessControl-AMC — APPROVED** [review]
Runbot OK. LGTM

**Review by carlosdauden — APPROVED** [review]

**Comment by pedrobaeza** [comment]
There's a weird commit: https://github.com/OCA/credit-control/pull/379/commits/9f0a42049a7dfe5b2261b706408985049e46ed15

squashing too many commits it seems and clearly with a wrong commit message. Can you please fix it?

**Comment by sbiosca-s73** [comment]
@pedrobaeza commit message updated

**Comment by pedrobaeza** [comment]
I think you should be put as co-author at least (or split into 2 commits), as right now the attribution is to the previous contributor, but he didn't do the hard work.

**Comment by sbiosca-s73** [comment]
@pedrobaeza done

**Pull request merged**